Dockerfile

```diff
@@ -8,4 +8,7 @@ COPY --from=builder /go/src/github.com/openshift/machine-api-operator/install ma
 COPY --from=builder /go/src/github.com/openshift/machine-api-operator/bin/machine-api-operator .
 COPY --from=builder /go/src/github.com/openshift/machine-api-operator/bin/nodelink-controller .
 COPY --from=builder /go/src/github.com/openshift/machine-api-operator/bin/machine-healthcheck .
+COPY --from=builder /go/src/github.com/openshift/machine-api-operator/bin/machineset ./manager
+COPY --from=builder /go/src/github.com/openshift/machine-api-operator/bin/vsphere ./machine-controller-manager
```

Comment:
This seems pretty confusing. Are we sure we don't want to build a separate image for the vSphere actuator? If the plan is to move more of these into this repo, we have to figure that out eventually anyway.

Reply:
I think I'd rather live with this oddity for now and get the benefits of it with no need to revendor at all for this provider. If we add another one in tree we'd could go (machine-controller-%v, platform) for naming the binaries

Reply:
Yeah, I can see the point about not having another image. I do think we should come up with some kind of wrapper that determines the platform and executes the correct controller / actuator though, but that can definitely wait until a follow-up.